cmd/wego/add/cmd.go

```diff
@@ -45,6 +45,7 @@ func init() {
 	Cmd.Flags().StringVar(&params.PrivateKey, "private-key", "", "Private key to access git repository over ssh")
 	Cmd.Flags().StringVar(&params.AppConfigUrl, "app-config-url", "", "URL of external repository (if any) which will hold automation manifests; NONE to store only in the cluster")
 	Cmd.Flags().BoolVar(&params.DryRun, "dry-run", false, "If set, 'wego add' will not make any changes to the system; it will just display the actions that would have been taken")
+	Cmd.Flags().BoolVar(&params.AutoMerge, "auto-merge", false, "If set, 'wego add' will merge automatically into the set --branch")
```

Comment:
auto merge seems right to me because we are skipping the PR stage and just merging right to the set branch. Auto push would make sense if this was for the PR. So if auto-merge is false then a PR is created.

Reply:
I don't feel strongly about changing this right now (particularly since we'll likely move to the config file stuff later). However, we "commit and push" in the non-PR path (any merging would have had to have already happened) :man_shrugging: